subprojects/core-api/src/main/java/org/gradle/api/artifacts/repositories/ArtifactRepository.java

```diff
@@ -15,13 +15,14 @@
  */
 package org.gradle.api.artifacts.repositories;
 
+import org.gradle.api.Describable;
 import org.gradle.internal.HasInternalProtocol;
 
 /**
  * A repository for resolving and publishing artifacts.
  */
 @HasInternalProtocol
-public interface ArtifactRepository {
+public interface ArtifactRepository extends Describable {
```

Comment:
⭕️ I'd probably put this on `ArtifactRepositoryInternal`, as we have no use case for users calling this at the moment.